src/Explorer/Panes/SettingsPane/SettingsPane.tsx

```diff
@@ -260,6 +280,29 @@ export const SettingsPane: FunctionComponent = () => {
             </div>
           </div>
         )}
+        {shouldShowPriorityLevelOption && (
```

Comment:
Why is this hidden behind a feature flag but not the actual sending of the header. It does not make sense to only hide the UI.